packages/cli/src/databases/migrations/common/1705928727784-CreateProject.ts

```diff
@@ -0,0 +1,45 @@
+import type { MigrationContext, ReversibleMigration } from '@db/types';
+
+const tableName = 'project';
+const relationTableName = 'project_relation';
+
+export class CreateProject1705928727784 implements ReversibleMigration {
+	async up({ dbType, schemaBuilder: { createTable, column } }: MigrationContext) {
+		await createTable(tableName).withColumns(
+			column('id').varchar(36).primary.notNull,
+			column('name').varchar(255),
+			column('type').varchar(36),
+		).withTimestamps;
+
+		let userIdColumn = column('userId').varchar().primary.notNull;
```

Comment:
could we use `uuid` for all DBs, and handle this in the DSL instead?

Reply:
The problem is `uuid` is an actual column type in SQLite as well (though I think it's basically a string under the hood?), it's just that the user `id` column is just different between the DBs. It's a `UUID` on postgres, a `varchar(36)` on mysql, and a `varchar` with no size limit on SQLite.

Reply:
I'd still prefer us to handle this in the DSL, and not add additional complexity in the migrations themselves.
I propose that we make [this change](https://github.com/n8n-io/n8n/commit/20f43ca8094358da6d7ed76eab88e2f2d21f5f3d) instead.